go.mod

```diff
@@ -0,0 +1,53 @@
+module code-review-bot
```

Comment:
Consider locking down versions more specifically rather than using tags like `v0.0.0-...`. This improves reproducibility of builds.